boards/arm/thingy52_nrf52832/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_GPIO)
+if(CONFIG_SENSOR)
```

Comment:
Actually, why not `if(CONFIG_CCS811)`?

Reply:
that's true since it's the ccs power rail, let me adjust